diff --git a/Dribbblish/color.ini b/Dribbblish/color.ini
index a425dd5..511ed61 100644
--- a/Dribbblish/color.ini
+++ b/Dribbblish/color.ini
@@ -306,3 +306,21 @@ tab-active         = 313244 ; Surface0
 notification       = 313244 ; Surface0
 notification-error = f38ba8 ; Red
 misc               = 45475a ; Surface1
+
+[onedark]
+text               = abb2bf
+subtext            = 5c6370
+sidebar-text       = abb2bf
+main               = 282c34
+sidebar            = 282c34
+player             = 282c34
+card               = 282c34
+shadow             = 282c34
+selected-row       = abb2bf
+button             = 5c6370
+button-active      = 98c379
+button-disabled    = 282c34
+tab-active         = 4b5263
+notification       = 282c34
+notification-error = e06c75
+misc               = abb2bf
